ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -35,6 +35,17 @@ then
     export GUEST_KUBECONFIG=${SHARED_DIR}/nested_kubeconfig
 fi
 
+# restore external oidc cache dir for oc
+if [[ -r "$SHARED_DIR/oc-oidc-token" ]] && [[ -r "$SHARED_DIR/oc-oidc-token-filename" ]]; then
+    echo "Restoring external OIDC cache dir for oc"
+    export KUBECACHEDIR
+    KUBECACHEDIR="/tmp/output"
+    token_cache_dir="$KUBECACHEDIR/oc"
```

Comment:
@fxierh 
better be `token_cache_dir="$KUBECACHEDIR/ocoidc"` because `oc` is too general

Reply:
Thanks for reviewing this PR. 

It appears to me that the sub-directory has to be `.../oc` which is due to the following logic in `oc`'s code:
https://github.com/openshift/oc/blob/8d46ebc7585c283cb6609e381e327912ce4318f4/pkg/cli/gettoken/gettoken.go#L123-L126

The `token_cache_dir` is used as a cache by the `oc` command, this could (in a way) explain the sub-directory name.

Reply:
how about 
```shell
    KUBECACHEDIR="/tmp/output/ocidc"
   mkdir -p $KUBECACHEDIR
    token_cache_dir="$KUBECACHEDIR/oc"
```
?

Reply:
Great idea. Rehearsing now.

Reply:
Rehearsal succeeded.